kobo/apps/trash_bin/utils/trash.py

```diff
@@ -221,7 +231,13 @@ def put_back(
         - `'attachment_uid'`
     """
 
-    trash_model, fk_field_name, related_model, *others = _get_settings(trash_type)
+    trash_model, fk_field_name, related_model, unique_identifier, *others = (
```

Comment:
I cannot comment on line 213, but let's use typing annotations to specify the return the value.
Also, what about using typing aliases to specify the type of `objects_list` too?